packages/react-native-sdk/src/utils/push/libs.ts

```diff
@@ -0,0 +1,50 @@
+export type { FirebaseMessagingTypes } from '@react-native-firebase/messaging';
+
+export type RNCallKeepType = typeof import('react-native-callkeep').default;
+export type FirebaseMessagingType =
+  typeof import('@react-native-firebase/messaging').default;
+export type VoipPushNotificationType =
+  typeof import('react-native-voip-push-notification').default;
+
+let callkeep: RNCallKeepType | undefined;
+let messaging: FirebaseMessagingType | undefined;
+let voipPushNotification: VoipPushNotificationType | undefined;
+
+try {
+  callkeep = require('react-native-callkeep').default;
+} catch (e) {}
+
+try {
+  messaging = require('@react-native-firebase/messaging').default;
+} catch (e) {}
+
+try {
+  voipPushNotification = require('react-native-voip-push-notification').default;
+} catch (e) {}
+
+export function getCallKeepLib() {
+  if (!callkeep) {
+    throw Error(
+      "react-native-callkeep library is not installed. Please install it using 'yarn add react-native-callkeep' or 'npm install react-native-callkeep'",
```

Comment:
Maybe here also send to https://github.com/react-native-webrtc/react-native-callkeep#Installation more future resistant